src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -67,7 +68,12 @@ public function addConfiguration(NodeBuilder $node): void
                     ->end()
                     ->scalarNode('audience')
                         ->info('Audience set in the token, for validation purpose.')
-                        ->defaultNull()
+                        ->isRequired()
+                    ->end()
+                    ->arrayNode('issuers')
+                        ->info('Issuers allowed to generate the token, for validation purpose.')
+                        ->isRequired()
+                        ->prototype('scalar')->end()
```

Comment:
I'd bet this means we should update the XSD also :)

Reply:
Where can I update it?

Reply:
src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

Reply:
src/Symfony/Bundle/SecurityBundle/Resources/config/schema/security-1.0.xsd

Reply:
Thanks.
The `access_token` authenticator is not declared in this file, I'm adding the complete configuration in it.

Reply:
should be fixed on 6.2 then

Reply:
A PR is in progress to update the XSD on 6.2 (adding `access-token`) and 6.3 (adding `oidc` and `oidc-user-info` over `access-token`).

Should we wait for these PRs before fixing this one?

Reply:
#50442 has just been merged.

Reply:
I don't see the other PR.

Reply:
Waiting for @aegypius